### PR TITLE
Release 2.5

### PR DESCRIPTION
**Pull request by ioguix**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
+  - add: new `oldest_xmin` service
+  - add: new `extensions_versions` service
+  - add: new `checksum_errors` service
+  - add: support for v13 and other improvements on `replication_slots`
+  - add: v13 compatibility for service `wal_files`
+  - add: various documentation details and examples
+  - add: support service `replication_slots` on standby
+  - add: accept single `b` or `o` as size unit
+  - add: json and json_strict output formats
+  - add: `size` and/or `delta` threshold for `database_size` service
```
Comment (Krysztophe) [inline CHANGELOG.md:15]: thresholds (pluriel) ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,7 +15,9 @@ Changelog
   - add: `size` and/or `delta` threshold for `database_size` service
   - add: thresholds are now optional for service `database_size`
   - add: support for v12 and v13 `archive_folder`
-  - regression: on service `replication_slots`: threshold `repslot` becomes `spilled`
+  - regression: threshold `repslot` becomes `spilled` in service `replication_slots`
+  - regression: a critical alert is now raised on table that were never analyzed/vacuumed
+    or lost maintenance date due to a crash in services `latest_vacuum` and `latest_analyze`
```
Comment (Krysztophe) [inline CHANGELOG.md:20]: in services `latest_vacuum` and `latest_analyze`:  a critical alert is now raised on tables that were never analyzed/vacuumed or whose maintenance date was lost due to a crash

File: CHANGELOG.md (hunk 1 above)
Comment (ioguix) [reply CHANGELOG.md:15]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (ioguix) [reply CHANGELOG.md:20]: thanks

**Pull request merged**